modules/manage/partials/backend-description-section.adoc

```diff
@@ -0,0 +1,4 @@
+[horizontal]
+*CouchStore*:: the regular mechanism suited to lower volumes of data.
+*Magma*:: the latest backend disk-only mechanism for high performance with much higher data volumes.
```

Comment:
No need to make definitions bold, our CSS does that anyway

Reply:
The glossary layout doesn't put the titles in bold if it's set to show as horizontal. Might need a CSS adjustment.

Reply:
Oh yes — in a horizontal layout, you are correct. Ignore this one!